.jshintignore

```diff
@@ -16,6 +16,7 @@ react/
 # The following are checked by ESLint with the minimum configuration which does
 # not supersede JSHint but take advantage of advanced language features such as
 # Facebook Flow which are not supported by JSHint.
+modules/analytics/analytics.js
```

Comment:
maybe it would be better to disable particular rule using inline comments instead of ignoring the whole file ?
example:
/* eslint-disable some-rule */
some ugly hacky code
/* eslint-enable some-rule */